localstripe/localstripe-v3.js

```diff
@@ -226,13 +226,13 @@ Stripe = (apiKey) => {
             payment_method_data: {
               type: 'card',
               card: {
-                number: element.value.number,
-                exp_month: element.value.exp_month,
-                exp_year: element.value.exp_year,
-                cvc: element.value.cvc,
+                number: data.payment_method.card.value.number,
```

Comment:
To keep code consistent, and to simplify, could we do the same as for SEPA?

https://github.com/adrienverge/localstripe/blob/c11d14df298cfb271473ad62534636000a29ba5a/localstripe/localstripe-v3.js#L324

Reply:
Hey Adrien!
So It's not possible to do that because when we follow [this way of using the API](https://stripe.com/docs/payments/save-and-reuse#add-elements-to-your-page) we have a 
```js
data.payment_method.card = {
   listeners: {change: [null]},
   values: {number: 12, exp_month:12, ...} `
```
And it's not directcly compatible with the API.
Will you be OK if I had a condition to check if the user used an object directly, or a Stripe Element?